### PR TITLE
Updated description, guidelines and example for gestational_age. Fixes #911.

### LINKED ISSUE #911: Correct gestational_age in the Donor schema
**For which schema is a change/update being suggested?**

I would like to request an update to the `donor_organism.json` schema.

**What should the change/update be?**

I would like to update the field description and guidelines for `gestational age`  as the current definition is incorrect.

**What new field(s) need to be changed/added?**

definition: Gestational age of pregnancy measured from the last menstrual period. 
guidelines: Add 2 weeks to embryonic post-ovulation age.

**Why is the change requested?**

The current description of gestational age is incorrect.

### PR DESCRIPTION
**Pull request by zperova**
### Release notes

For donor_organism schema:
- Corrected description for gestational_age.
- Updated guidelines and examples for gestational_age. 
 

### Reviews requested

- Need 1 Reviewer to approve because this is a patch update
- Last Reviewer to review/approve, merge changes by 19 April.

### CODE REVIEW

**Comment by zperova** [comment]
it was my "no update_log" day it seems. @malloryfreeberg done now

File: json_schema/update_log.csv [hunk 1]
```diff
@@ -1 +1,2 @@
 Schema,Change type,Change message,Version,Date
+type/donor_organism/gestational_age,patch,"Changed description, guidelines and example for gestational_age. Fixes #911.",,
```
Comment (malloryfreeberg) [inline json_schema/update_log.csv]: The schema name isn't right. It needs `biomaterial` after type and shouldn't include the field name.

Comment (zperova) [reply json_schema/update_log.csv]: sorry, that was a silly mistake.

Comment (zperova) [reply json_schema/update_log.csv]: @malloryfreeberg

**Pull request merged**